thesis-de/content-de.tex

```diff
@@ -1888,16 +1888,104 @@ \section{Userstories}
 Tickets arbeiten können.
 \tm%
 
-\subsection{Userstory 1}
+\subsection{SSD Userstory}
+\textbf{Name der Story:} Anzeige der SSD's, welche einen Hardwaredeffekt haben könnten.
```

Comment:
der name stimmt so nicht. stories werden immer aus sicht des anwenders geschrieben.
Mehr infos: http://www.ksimons.de/2015/03/user-stories-verstaendlich-erklaert/

in dem fall: As a system adminsitrator I would like to know if I kill my SSDs (aus https://jira.virtapi.org/browse/DCC-45)